views/works.htm

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/works.css">
+    <title>Document</title>
+</head>
+<body>
+    
+    <h1>Catch Me If You Can</h1>
+
+    <img src="../public/images/catchme.png">
```

Comment:
Remember that in app.js we already sais that we can access all files that are in public folder `app.use(express.static('public'));`

So we do not need to write public when adding link for image:
`<img src="/images/catchme.png">`